distutils/tests/test_sysconfig.py

```diff
@@ -8,17 +8,21 @@
 import pytest
 import jaraco.envs
 import path
-from jaraco.text import trim
```

Comment:
Just wondering, which dependency was the problem? (I guess I could try)

Either way it seems like a good trade to lose a dependency, but ymmv of course.

Reply:
ah, pydantic-core gets pulled in. It builds fine here if I install rust, but it takes quite a bit to build (some minutes)